ci/src/datakit_ci.mli

```diff
@@ -0,0 +1,664 @@
+(** Datakit CI
+
+    [Datakit_ci] is a library to describe reproducible, cachable and
+    distributed computations. The library is specialiazed to handle
+    build pipelines, where computations can take time, and where
+    seeing the live outputs of the computation is as useful as the
+    final step (think of live-logs of long-running builds).
+
+    The {!core} of [Datakit_ci] are {{!Term}term}combinators to
+    describe the composition of long-running computation jobs, with an
+    attached, individual {{!Output}output} (which can be
+    {{!livelogs}live}). A project {{!config}configuration} tied
+    together a collection of job description and a simple description
+    for the {{!web}web} interface, whose access is granted using
+    {{!acl}ACLs}.
+
+    TODO.
+*)
+
+(** {1:core Core} *)
+
+module DK: Datakit_S.CLIENT with type error = Protocol_9p_error.error
```

Comment:
All these formatting changes (" : " to ": ") waste a lot of reviewing time. Can we just keep the style the project is already using? Especially for code that already exists - there's no change here except to replace one coding style with a slightly different one.